internal/network_test_utils.go

```diff
@@ -115,7 +116,7 @@ func (l localNetworkServices) Serve(ctx context.Context) error {
 		for path, contents := range l.webpages {
 			webpage := contents
 			http.HandleFunc("/"+path, func(w http.ResponseWriter, r *http.Request) {
-				fmt.Fprintf(w, webpage)
+				io.WriteString(w, webpage)
```

Comment:
out of curiosity, why change this?

Reply:
golangci-lint is not happy about using `*f` function for just writing a string.